### PR TITLE
Smallfile dockerfile

### PR DESCRIPTION
**Pull request by bengland2**
This is the first time I've run smallfile inside an AWS cluster successfully.   I had to start trying out run_snafu.py inside a container built with smallfile_wrapper/Dockerfile.   A corresponding commit in ripsaw will be coming shortly, but you can test this on your local laptop by just building the container and running run_snafu.py within it.

### CODE REVIEW

File: smallfile_wrapper/Dockerfile [hunk 1]
```diff
@@ -1,10 +1,14 @@
...
+RUN pip install --upgrade pip
+RUN pip install elasticsearch statistics
 RUN git clone https://github.com/distributed-system-analysis/smallfile
-RUN mv smallfile smallfile-master
-RUN mkdir -p /opt/snafu/
-COPY . /opt/snafu/
+RUN mv smallfile /opt/
+RUN ln -sv /opt/smallfile/smallfile_cli.py /usr/local/bin/
+RUN ln -sv /opt/smallfile/smallfile_rsptimes_stats.py /usr/local/bin/
+RUN git clone https://github.com/bengland2/snafu
```
Comment (aakarshg) [inline smallfile_wrapper/Dockerfile]: why are using the fork here?

Comment (bengland2) [reply smallfile_wrapper/Dockerfile]: oops will fix, I meant to just use that for development purposes not in the actual commit.

**Comment by bengland2** [comment]
better now?

File: smallfile_wrapper/Dockerfile [hunk 2]
```diff
@@ -1,10 +1,16 @@
...
+RUN pip install --upgrade pip
+RUN pip install elasticsearch statistics
 RUN git clone https://github.com/distributed-system-analysis/smallfile
-RUN mv smallfile smallfile-master
-RUN mkdir -p /opt/snafu/
-COPY . /opt/snafu/
+RUN mv smallfile /opt/
+RUN ln -sv /opt/smallfile/smallfile_cli.py /usr/local/bin/
+RUN ln -sv /opt/smallfile/smallfile_rsptimes_stats.py /usr/local/bin/
+RUN git clone https://github.com/cloud-bulldozer/snafu
```
Comment (aakarshg) [inline smallfile_wrapper/Dockerfile]: cloning isn't necessary, the original way of doing a copy like:

```
RUN mkdir -p /opt/snafu/	RUN ln -sv /opt/smallfile/smallfile_cli.py /usr/local/bin/
COPY . /opt/snafu/
```

Comment (bengland2) [reply smallfile_wrapper/Dockerfile]: ok this version does do COPY instead of clone, I didn't know how image was being built until today.

**Pull request merged**